tests/functional/test_data_completeness.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from botocore.session import Session
+from botocore.loaders import Loader
+from botocore.exceptions import DataNotFoundError
+
+
+class TestDataTypes(object):
```

Comment:
+1 on keeping it consistent with botocore. All generator-driven tests so far have been expressed as functions and not methods of classes. Furthermore, the class you created does not inherit from ``unittest.TestCase`` so you are not taking advantage of any of the classes testing methods like ``setUp``, ``tearDown`` or any of the assertions. So, there is no *current* reason to wrap the tests in a class.